src/Routes.tsx

```diff
@@ -1,15 +1,18 @@
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import JoinRoom from "./routes/JoinRoom";
+import LandingPage from "./routes/LandingPage";
+import CreateRoomPage from "./routes/CreateRoomPage";
 
 const AppRoutes = () => {
-    return (
-        <Router>
-            <Routes>
-                <Route path="/" element={<h1>Hello !Toohak</h1>} />
-                <Route path="/join/:room_id" element={<JoinRoom />} />
-            </Routes>
-        </Router>
-    );
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="/create-room" element={<CreateRoomPage />} />
```

Comment:
[NIT] Call the component `CreateRoom` instead of `CreateRoomPage` to adhere to the convention we established with `JoinRoom`